demos/bookstore/app/middleware/auth.ts

```diff
@@ -1,5 +1,5 @@
-import type { Middleware, Route } from '@remix-run/fetch-router'
-import { createRedirectResponse as redirect } from '@remix-run/response/redirect'
+import type { Middleware, Route } from 'remix'
+import { createRedirectResponse as redirect } from 'remix'
```

Comment:
This one is so common we'll probably want to have a `redirect` shorthand that people can import instead of renaming it every time.

Reply:
Do you think we should do the same for `createHtmlResponse` and `createFileResponse`?  `html` would conflict at a global level with the `html` string helper so if we chose to export from the top-level we'd have to choose one, but they'd both be fine as exports from their individual sub-exports.

Reply:
https://github.com/remix-run/remix/pull/10894/commits/1210a547c786bba678f9408c7932a98175d93367

Reply:
I'm not sure about the `html` helper. I want to see how it feels once we start using components more heavily in the demos and relying less on the html-template library. I just know for now that I want to be able to import `redirect` directly, or I'm always going to do this rename.